### PR TITLE
UX font tweaks

### PR DESCRIPTION
**Pull request by mistermendez**
merge this into `pbw-3634-screens` before merging `pbw-3634-screens` into master

### CODE REVIEW

File: js/views/pauseScreen.js [hunk 1]
```diff
@@ -54,13 +54,13 @@ var PauseScreen = React.createClass({
   render: function() {
     //inline style for config/skin.json elements only
     var titleStyle = {
-      fontSize: this.props.skinConfig.startScreen.titleFont.fontSize + "pt",
-      fontFamily: this.props.skinConfig.startScreen.titleFont.fontFamily,
+      //fontSize: this.props.skinConfig.startScreen.titleFont.fontSize + "pt",
```
Comment (mistermendez) [inline js/views/pauseScreen.js]: temporarily disabling this until we sync font sizes with PBA

File: js/views/pauseScreen.js [hunk 2]
```diff
@@ -54,13 +54,13 @@ var PauseScreen = React.createClass({
...
+      //fontFamily: this.props.skinConfig.startScreen.titleFont.fontFamily,
       color: this.props.skinConfig.startScreen.titleFont.color
     };
     var descriptionStyle = {
-      fontSize: this.props.skinConfig.startScreen.descriptionFont.fontSize + "pt",
-      fontFamily: this.props.skinConfig.startScreen.descriptionFont.fontFamily,
+      //fontSize: this.props.skinConfig.startScreen.descriptionFont.fontSize + "pt",
```
Comment (mistermendez) [inline js/views/pauseScreen.js]: temporarily disabling this until we sync font sizes with PBA

File: js/views/startScreen.js [hunk 3]
```diff
@@ -81,13 +81,13 @@ var StartScreen = React.createClass({
   render: function() {
     //inline style for config/skin.json elements only
     var titleStyle = {
-      fontSize: this.props.skinConfig.startScreen.titleFont.fontSize + "pt",
-      fontFamily: this.props.skinConfig.startScreen.titleFont.fontFamily,
+      //fontSize: this.props.skinConfig.startScreen.titleFont.fontSize + "pt",
```
Comment (mistermendez) [inline js/views/startScreen.js]: temporarily disabling this until we sync font sizes with PBA

**Comment by chibzu** [comment]
Looks good.

**Pull request merged**